agcaballero/portfolio/pom.xml

```diff
@@ -23,6 +23,12 @@
       <version>4.0.1</version>
       <scope>provided</scope>
     </dependency>
+
+    <dependency>
+      <groupId>com.google.code.gson</groupId>
+      <artifactId>gson</artifactId>
+      <version>2.8.6</version>
+    </dependency>
```

Comment:
I think I also saw this in a different PR, should this PR have a different base branch?

Reply:
I don't think so, especially now that the comments branch commits are closed completely.